src/main/java/edu/iis/powp/adapter/AbstractPlotterAdapter.java

```diff
@@ -0,0 +1,26 @@
+package edu.iis.powp.adapter;
+
+import edu.iis.client.plottermagic.AbstractPlotter;
+import edu.iis.client.plottermagic.IPlotter;
+
+public class AbstractPlotterAdapter extends AbstractPlotter {
+    IPlotter plotter;
+
+    public AbstractPlotterAdapter(IPlotter iPlotter) {
+        super(0, 0);
+        this.plotter = iPlotter;
+    }
+
+    @Override
+    public void drawTo(int x, int y)
+    {
+        this.plotter.drawTo(x, y);
```

Comment:
drawTo is implemented incorrectly.
plotter instance is not updated when setPosition is called on AbstractPlotterAdapter. (TIP: use AbstractPlotter getX and getY to update before calling drawTo).